### PR TITLE
Dbaas review

### PR DESCRIPTION
**Pull request by bavedarnow**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -3051,29 +3051,27 @@ paths:
...
+        source: >
+          linode-cli databases version-view mysql/5.7.30
   /databases/instances:
     x-linode-cli-command: databases
     get:
       tags:
         - Databases
       summary: Managed Databases List
       operationId: getDatabasesInstances
-      servers:
```
Comment (bbiggerr) [inline openapi.yaml]: any reason this is removed? should only be accessible at v4beta, and this designates the command as in beta

File: openapi.yaml [hunk 2]
```diff
@@ -3177,13 +3171,7 @@ paths:
         content:
           application/json:
             schema:
-              required:
```
Comment (bbiggerr) [inline openapi.yaml]: any reason the required block is removed here? it can be moved into the schema instead

File: openapi.yaml [hunk 3]
```diff
@@ -17073,48 +17039,44 @@ components:
       properties:
         id:
           $ref: '#/components/schemas/Database/properties/id'
-          x-linode-cli-display: 1
```
Comment (bbiggerr) [inline openapi.yaml]: extension redefinitions required here--CLI doesn't pick them up from the $refs

File: openapi.yaml [hunk 4]
```diff
@@ -17169,7 +17164,7 @@ components:
       properties:
         ca_certificate:
           type: string
-          description: The SSL CA certificate for the Managed Database instance.
+          description: The SSL CA certificate for the Managed Database instance. (Mention it's base64 encoded)
```
Comment (bavedarnow) [inline openapi.yaml:17167]: Pointing out this

File: openapi.yaml (hunk 2 above)
Comment (bavedarnow) [reply openapi.yaml]: Moved it to the schema

**Pull request closed without merging**